Exercise 2/Start/BackgroundDownload/Info.plist

```diff
@@ -1,59 +1,61 @@
-    <key>MinimumOSVersion</key>
-    <string>8.0</string>
-    <key>UIDeviceFamily</key>
-    <array>
-      <integer>1</integer>
-    </array>
-    <key>UIMainStoryboardFile</key>
-    <string>MainStoryboard</string>
-    <key>UIRequiredDeviceCapabilities</key>
-    <array>
-      <string>armv7</string>
-    </array>
-    <key>UISupportedInterfaceOrientations</key>
-    <array>
-      <string>UIInterfaceOrientationPortrait</string>
-    </array>
-    <key>UIBackgroundModes</key>
-    <array>
-      <string>audio</string>
-    </array>
-    <key>CFBundleDisplayName</key>
-    <string>BG Download</string>
-    <key>XSAppIconAssets</key>
-    <string>Resources/Images.xcassets/AppIcons.appiconset</string>
-    <key>XSLaunchImageAssets</key>
-    <string>Resources/Images.xcassets/LaunchImage.launchimage</string>
-    <key>UILaunchStoryboardName</key>
-    <string>LaunchScreen</string>
-    <key>NSAppTransportSecurity</key>
-    <dict>
-      <key>NSExceptionDomains</key>
-      <dict>
-        <key>xamarinuniversity.blob.core.windows.net</key>
-        <dict>
-          <key>NSExceptionMinimumTLSVersion</key>
-          <string>TLSv1.0</string>
-          <key>NSExceptionRequiresForwardSecrecy</key>
-          <false />
-          <key>NSExceptionAllowsInsecureHTTPLoads</key>
-          <true />
-          <key>NSIncludesSubdomains</key>
-          <true />
-        </dict>
-      </dict>
-    </dict>
-  </dict>
-</plist>
\ No newline at end of file
+<dict>
+	<key>CFBundleIdentifier</key>
+	<string>com.xamarin.backgrounddownload</string>
+	<key>CFBundleShortVersionString</key>
+	<string>1.0</string>
+	<key>CFBundleVersion</key>
+	<string>1.0</string>
+	<key>LSRequiresIPhoneOS</key>
+	<true/>
+	<key>MinimumOSVersion</key>
+	<string>8.0</string>
+	<key>UIDeviceFamily</key>
+	<array>
+		<integer>1</integer>
+	</array>
+	<key>UIMainStoryboardFile</key>
+	<string>MainStoryboard</string>
+	<key>UIRequiredDeviceCapabilities</key>
+	<array>
+		<string>armv7</string>
+	</array>
+	<key>UISupportedInterfaceOrientations</key>
+	<array>
+		<string>UIInterfaceOrientationPortrait</string>
+	</array>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>audio</string>
+	</array>
+	<key>CFBundleDisplayName</key>
+	<string>BG Download</string>
+	<key>XSAppIconAssets</key>
+	<string>Resources/Images.xcassets/AppIcons.appiconset</string>
+	<key>XSLaunchImageAssets</key>
+	<string>Resources/Images.xcassets/LaunchImage.launchimage</string>
+	<key>UILaunchStoryboardName</key>
+	<string>LaunchScreen</string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>xamarinuniversity.blob.core.windows.net</key>
+			<dict>
+				<key>NSExceptionMinimumTLSVersion</key>
+				<string>TLSv1.0</string>
+				<key>NSExceptionRequiresForwardSecrecy</key>
+				<false/>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+				<key>NSIncludesSubdomains</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
+	<key>CFBundleName</key>
```

Comment:
This `CFBundleName` was a change made in the mass of whitespace changes. Since it probably doesn't hurt to have it in Completed, I'll copy it there as well.